lib/vquic/ngtcp2.c

```diff
@@ -980,7 +1036,13 @@ static ssize_t ngh3_stream_recv(struct connectdata *conn,
     stream->memlen = 0;
     stream->mem = buf;
     stream->len = buffersize;
+    /* extend the stream window with the data we're consuming and send out
```

Comment:
This was missing from the change in #4600. This was causing the `MAX_STREAM_DATA` frames that `extend_stream_window` was generating to not be sent right away. If the server had already reached the flow control limit we would never receive the next packet from them and we'd never actually send these frames, so the transfer would hang.